packages/remark-grid-tables/package.json

```diff
@@ -33,8 +33,9 @@
   "license": "MIT",
   "dependencies": {
     "grapheme-splitter": "^1.0.4",
+    "is-fullwidth-code-point": "^3.0.0",
```

Comment:
Extraneous dep: not actually used in this package. That's just a tiny merge mistake.